l10n_fr_siret_lookup/__manifest__.py

```diff
@@ -14,7 +14,6 @@
     "depends": [
         "l10n_fr_siret",
     ],
-    "external_dependencies": {"python": ["requests", "python-stdnum>=1.18"]},
```

Comment:
It's okay to specify direct dependencies here, if only for docs purposes. So the line must not be removed.

Maybe there a good reason to depend on python-stdnum>1.18 for this module to work? In such case we must accept it.

Reply:
> So the line must not be removed.

Restored

Reply:
Hi @yajo thank you for looking into this,
as a matter of fact there is a good reason to request python-stdnum >= 1.18, which is this commit : https://github.com/arthurdejong/python-stdnum/commit/73f5e3a864915e9f3f2b01683ab9b192b4217e46

The module should work with older version, although it will fail for the specific SIRET covered by the above commit (but this specific case is not tested in the module so unit tests should not fail neither if done with older version)